jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/Annotator.java

```diff
@@ -73,8 +73,10 @@
      *            JSON property
      * @param propertyName
      *            the name of the JSON property that this getter gets
+     * @param propertyNode
+     *            the schema node defining this property
      */
-    void propertyGetter(JMethod getter, String propertyName);
+    void propertyGetter(JMethod getter, String propertyName, JsonNode propertyNode);
```

Comment:
This change alters the existing contract for Annotator and will break existing custom annotators.  This change should be additive.

Reply:
```suggestion
    default void propertyGetter(JMethod getter, String propertyName, JsonNode propertyNode) {
        propertyGetter(getter, propertyName);
    }
```

@thachhoang with this, we can call the new overload from the rule classes. Existing custom annotators will still get the the expected parameters delivered, but the jackson annotator can override the new propertyGetter and make use of the propertyNode.